osmaxx-py/excerptconverter/tests/test_gis_excerpt_converter.py

```diff
@@ -0,0 +1,5 @@
+from django.test.testcases import TestCase
+
+
+class GisExcerptConverterTestCase(TestCase):
+    pass
```

Comment:
Should we revive the test from https://github.com/geometalab/osmaxx/commit/e55699d9566bfa004557c6b68cb251014084c973#diff-49923a992fdcc1df46f58199ea842a64L5 and add some more?